src/drawable.h

```diff
@@ -157,4 +137,30 @@ inline void Drawable::SetVisible(bool value) {
 	_flags = value ? _flags & ~Flags::Invisible : _flags | Flags::Invisible;
 }
 
+static constexpr uint64_t z_offset = 64 - 8;
```

Comment:
magic number: comment about 8 bit tile cache?

Reply:
This is unrelated to the tile cache. It is used below by the layers to enforce the right rendering order (Is the same as the hardcoded ``<< 24`` in the old code, just now for 64 bit instead of 32)